[BUG] Two-Factor Authentication Required on host configured with Authentication None

### Title

[BUG] Two-Factor Authentication Required on host configured with Authentication None

### Platform

Website - Firefox

### Server Installation Method

Docker

### Version

1.11.1

### Troubleshooting

- [x] I have examined logs and tried to find the issue
- [x] I have reviewed opened and closed issues
- [x] I have tried restarting the application
- [x] I have read Known Issues (https://github.com/Termix-SSH/Support/issues/296)

### The Problem

This is a fresh 1.11.1 installation. The first host I added seems to work, but the second one keeps popping up the "Two-Factor Authentication Required" dialogue, even though authentication is set to None on both. 
I am able to connect to this host using password authentication from my local PC terminal.

<img width="472" height="228" alt="Image" src="https://github.com/user-attachments/assets/7e9d3ffc-d809-439a-9d32-0c2e5921346f" />

I have attempted:
- connecting from Firefox on two different Linux PCs, and Chromium from one with the same results. 
- both connecting with hostname and IP address. 
- toggling Force Keyboard-Interactive
- clearing browser cookies and site data
- restarting the docker container

The only lines I see in the log that seem relevant to the specific connection attempt are:
```
[termix] 2026-02-15T15:09:10.406942716Z [3:09:10 PM] [INFO] [🖥️] Terminal WebSocket connection established [op:terminal_ws_connect,user:8pD5pkuoidZWd_TMZPWOV,session:bvMUoMPLtJmF4ZwA1soOM]
[termix] 2026-02-15T15:09:10.411950033Z [3:09:10 PM] [INFO] [🖥️] Resolving SSH host configuration [op:terminal_host_resolve,user:8pD5pkuoidZWd_TMZPWOV,host:2,session:bvMUoMPLtJmF4ZwA1soOM]
[termix] 2026-02-15T15:09:10.412780525Z [3:09:10 PM] [INFO] [🖥️] Initiating SSH connection [op:terminal_ssh_connect_attempt,user:8pD5pkuoidZWd_TMZPWOV,host:2,session:bvMUoMPLtJmF4ZwA1soOM]
[termix] 2026-02-15T15:09:10.743984807Z [3:09:10 PM] [INFO] [🖥️] Host key verified successfully [op:host_key_verified,user:8pD5pkuoidZWd_TMZPWOV,host:2]
[termix] 2026-02-15T15:11:10.417266878Z [3:11:10 PM] [ERROR] [🖥️] SSH connection error [op:ssh_connect,host:2]
[termix] 2026-02-15T15:11:10.417802426Z Error: Timed out while waiting for handshake
[termix] 2026-02-15T15:11:10.417825085Z     at Timeout._onTimeout (/app/node_modules/ssh2/lib/client.js:1114:23)
[termix] 2026-02-15T15:11:10.417832426Z     at listOnTimeout (node:internal/timers:588:17)
[termix] 2026-02-15T15:11:10.418075093Z     at process.processTimers (node:internal/timers:523:7) {
[termix] 2026-02-15T15:11:10.418086803Z   level: 'client-timeout'
[termix] 2026-02-15T15:11:10.418092891Z }
[termix] 2026-02-15T15:11:10.418385891Z [3:11:10 PM] [ERROR] [🖥️] Proceeding with cleanup after error
[termix] 2026-02-15T15:11:10.418402264Z {
[termix] 2026-02-15T15:11:10.418408505Z   operation: 'ssh_error_cleanup',
[termix] 2026-02-15T15:11:10.418414359Z   hostId: 2,
[termix] 2026-02-15T15:11:10.418420039Z   error: 'Timed out while waiting for handshake'
[termix] 2026-02-15T15:11:10.418425637Z }
[termix] 2026-02-15T15:11:10.419751083Z [3:11:10 PM] [INFO] [🖥️] SSH connection closed [op:terminal_ssh_disconnected,user:8pD5pkuoidZWd_TMZPWOV,host:2,session:bvMUoMPLtJmF4ZwA1soOM]
```

The in application connection log shows:
```
[10:09:10 AM] [INFO] Starting address resolution of sam-cb-pro.domain.localnet
[10:09:10 AM] [INFO] Connecting to sam-cb-pro.domain.localnet port 22
[10:09:10 AM] [INFO] Starting SSH session
[10:09:10 AM] [INFO] Authenticating as cory
[10:09:10 AM] [INFO] Password authentication required
[10:11:10 AM] [ERROR] Connection error: Timed out while waiting for handshake
[10:11:10 AM] [ERROR] Connection failed: Timed out while waiting for handshake
[10:11:10 AM] [ERROR] SSH error: Timed out while waiting for handshake
[10:12:10 AM] [ERROR] Connection rejected by server. Please check your authentication and network configuration.
```

### How to Reproduce

Set up the host with minimal configuration:
- Add IP Address/Hostname
- Username
- Host name
- Set Authentication to None

Even adding a new host with the same address and username asks for two-factor authentication.

### Additional Context

_No response_

--- a/README.md
+++ b/README.md
@@ -169,6 +169,10 @@ networks:
   <a href="https://akamai.com/">
     <img src="https://upload.wikimedia.org/wikipedia/commons/8/8b/Akamai_logo.svg" height="50" alt="Akamai">
   </a>
+  &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
+  <a href="https://aws.amazon.com/">
+    <img src="https://upload.wikimedia.org/wikipedia/commons/thumb/9/93/Amazon_Web_Services_Logo.svg/960px-Amazon_Web_Services_Logo.svg.png" height="50" alt="AWS">
+  </a>
 </p>
 
 # Support

--- a/src/backend/ssh/terminal.ts
+++ b/src/backend/ssh/terminal.ts
@@ -255,7 +255,7 @@ async function createJumpHostChain(
           host: jumpHostConfig.ip?.replace(/^\[|\]$/g, "") || jumpHostConfig.ip,
           port: jumpHostConfig.port || 22,
           username: jumpHostConfig.username,
-          tryKeyboard: true,
+          tryKeyboard: jumpHostConfig.authType !== "none",
           readyTimeout: 30000,
           hostVerifier: jumpHostVerifier,
         };
@@ -1801,7 +1801,7 @@ wss.on("connection", async (ws: WebSocket, req) => {
       host: ip,
       port,
       username,
-      tryKeyboard: true,
+      tryKeyboard: resolvedCredentials.authType !== "none",
       keepaliveInterval:
         typeof hostKeepaliveInterval === "number"
           ? hostKeepaliveInterval

